oop-essentials/ide-features/README.md

```diff
@@ -140,7 +140,7 @@ When jumping between files, being able to quickly return to a previous file (wit
 | IDE       | Shortcut       |
 | --------- |----------------|
 | JetBrains | Ctrl+E (Cmd+E) |
-| VSCode    | TODO           |
+| VSCode    | Ctrl+Alt+Left/Right (Cmd+Opt+Left/Right) |
```

Comment:
This is probably some other navigation. I believe the recent editors are also Ctrl+E in VSCode?

Reply:
ah damn yes this is to navigate back and forth to files. cmd+P allows you to search and see all the recently used files. Let me change this